test/deals.js

```diff
@@ -1,11 +1,73 @@
-const chai = require('chai')
-const expect = chai.expect
-
+const { expect } = require('chai')
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: 'demo' })
+const fakeHubspotApi = require('./helpers/fake_hubspot_api')
 
 describe('Deals', function() {
+  const hubspot = new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'fake-token',
+  })
+  const dealProperties = [
+    {
+      value: 'MadKudu',
+      name: 'dealname',
+    },
+    {
+      value: 'appointmentscheduled',
+      name: 'dealstage',
+    },
+    {
+      value: 'default',
+      name: 'pipeline',
+    },
+    {
+      value: Date.now(),
+      name: 'closedate',
+    },
+    {
+      value: '60000',
+      name: 'amount',
+    },
+    {
+      value: 'newbusiness',
+      name: 'dealtype',
+    },
+  ]
+  const companyProperties = [
+    { name: 'name', value: 'A company name' },
+    { name: 'description', value: 'A company description' },
+  ]
+  const createTestDeal = () =>
+    hubspot.deals.create({ properties: dealProperties })
+  const deleteTestDeal = dealId => hubspot.deals.deleteById(dealId)
+  const createTestCompany = () =>
+    hubspot.companies.create({ properties: companyProperties })
+  const createDealWithCompany = () =>
+    hubspot.companies
+      .create({ properties: companyProperties })
+      .then(companyData => {
+        return hubspot.deals
+          .create({
+            associations: {
+              associatedCompanyIds: [companyData.companyId],
+            },
+            properties: dealProperties,
+          })
+          .then(dealData => {
+            return {
+              dataCompanyId: companyData.companyId,
+              dataDealId: dealData.dealId,
+            }
+          })
+      })
+  const deleteTestCompany = companyId => hubspot.companies.delete(companyId)
+
   describe('get', function() {
+    const dealsEndpoint = {
+      path: '/deals/v1/deal/paged',
+      response: { deals: [] },
```

Comment:
Since this path includes the term "paged", does this response return a `has_more` key?

Reply:
It does. I've been tending not to test for it. I'll remove the assertions I have on hasMore in the other test.